### PR TITLE
update: the 'request a new extension' section

### PR DESCRIPTION
**Pull request by joshuabeckett**
## Describe your changes
I updated the [documentation](https://aiven.io/docs/products/postgresql/howto/manage-extensions#request-a-new-extension) that stated how to request a new extension, to be more accurate. The customer should do this through the Ideas Portal, not create a support ticket, since we just divert them to that anyway

JIRA ticket
https://aiven.atlassian.net/browse/SRE-8655

### CODE REVIEW

File: docs/products/postgresql/howto/manage-extensions.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ extensions are safe to upgrade.
 We are always open to suggestions of additional extensions that could be
 useful to many of our customers, and there are a few that can be enabled
 on request if you need them. For any extensions not on the
-[Extensions on Aiven for PostgreSQL®](/docs/products/postgresql/reference/list-of-extensions) approved list, please open a support ticket and let us know:
+[Extensions on Aiven for PostgreSQL®](/docs/products/postgresql/reference/list-of-extensions) approved list, please make a request through our [Ideas Portal](https://ideas.aiven.io//). Be sure to include:
```
Comment (staceysalamon-aiven) [inline docs/products/postgresql/howto/manage-extensions.md:43]:
```suggestion
[list of approved extensions](/docs/products/postgresql/reference/list-of-extensions), make a request through [Aiven Ideas](https://ideas.aiven.io/). Be sure to include:
```

**Review by staceysalamon-aiven — APPROVED** [review]
Thank you for updating this! I just made a minor change to use the name of the site, Aiven Ideas, and reworded this slightly to fit with our style guide. :)

File: docs/products/postgresql/howto/manage-extensions.md (hunk 1 above)
Comment (staceysalamon-aiven) [inline docs/products/postgresql/howto/manage-extensions.md:43]:
```suggestion
[list of approved extensions](/docs/products/postgresql/reference/list-of-extensions), make a request through [Aiven Ideas](https://ideas.aiven.io/). Be sure to include:
```

**Review by staceysalamon-aiven — CHANGES_REQUESTED** [review]

**Review by joshuabeckett — COMMENTED** [review]
LGTM!

**Review by staceysalamon-aiven — APPROVED** [review]

**Pull request merged**